README.md

```diff
@@ -54,3 +55,51 @@ This repository implments tests using [pytest](https://docs.pytest.org/), the te
 
 [^env_vars]: [For linux](https://www.gnu.org/software/bash/manual/bash.html#Environment) you can either prepend as in `NAME=value pipenv ...` or use `export NAME=value`, [for windows](https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/set_1) you can use `set`.
 [^user_config]: A more detailed documentation on [the user configuration file can be found here](docs/user_configuration_file.md).
+
+## AWS Packing
+Since some services from infin-neat-core code runs at AWS cloud services, it's necessary to properly pack the code to upload it using the web interface, AWS CLI, or AWS CDK. 
```

Comment:
I'm going to suggest a new version of this paragraph:


> Since some services from infin-neat-core code runs at AWS cloud services, it's necessary to properly pack the code to upload it using the web interface, AWS CLI, or AWS CDK.

In-fin-neat-core code runs on AWS Lambda and a "packing" command is defined generating a zip file with python dependencies (defined in Pipfile) installed to run on the AWS Lambda runtime.

This packing command uses the AWS Lambda docker container to install dependencies, as opposed to [aws official recommendation](https://docs.aws.amazon.com/lambda/latest/dg/python-package.html) which uses virtual environments. This is because <add your reason here, simpler, straightforward, etc..>

Reply:
I have tested again against the official manual: https://docs.aws.amazon.com/lambda/latest/dg/python-package.html. It runs well, defining the manylinux2014_x86_64 architecture. In this way, I could remove the dependency of Docker. So it's not necessary to install it anymore